pkg/assets/asset.go

```diff
@@ -0,0 +1,78 @@
+package assets
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"text/template"
+)
+
+var (
+	AssetPathBootstrapKubeAPIServer = "bootstrap-manifests/bootstrap-apiserver.yaml"
+)
+
+// TODO: Move to library-go
```

Comment:
@deads2k if we choose templates, i will move this to library-go and make it common